Interpreter.py

```diff
@@ -64,13 +65,23 @@ def brainfuck(program):
     if data_pointer != 0:
         print("WARNING (interpreter) - at the end of the execution the data pointer is %s instead of 0 (possibly a compiler issue)" % str(data_pointer))
 
+
 if __name__ == '__main__':
-    if len(sys.argv) != 2:
-        print("Usage: '%s' <path_to_brainfuck_code_file>" % sys.argv[0])
+    if len(sys.argv) < 2:
```

Comment:
This check can be removed, as the argparse library deals it for us :)

Reply:
Life was never easier :)